### PR TITLE
CVS-133383 Update models used in nightly/stress tests

### PR DESCRIPTION
**Pull request by wkobielx**
Old models listed in configs were removed from omz repository

### Tickets:
 - CVS-133383
 
Models changed basing on docs and availability in [omz_repo](https://github.com/openvinotoolkit/open_model_zoo/tree/master/models)
- https://docs.openvino.ai/nightly/omz_demos_classification_demo_python.html
- https://docs.openvino.ai/nightly/omz_demos_face_recognition_demo_python.html
- https://docs.openvino.ai/nightly/omz_demos_classification_demo_python.html
- https://docs.openvino.ai/nightly/omz_demos_single_human_pose_estimation_demo_python.html

### CODE REVIEW

**Comment by wkobielx** [comment]
Building ubuntu20 artifacts: https://openvino-ci.toolbox.iotg.sclab.intel.com/job/private-ci/job/ie/job/build-linux-ubuntu20/187887/

File: tests/stress_tests/.automation/memcheck_tests/nightly_configs/desktop_test_config.xml [hunk 1]
```diff
@@ -6,77 +6,75 @@
...
         <model name="ctpn" precision="FP32" source="omz" />
         <model name="i3d-rgb-tf" precision="FP32" source="omz" />
-        <model name="mobilenet-v2" precision="FP32" source="omz" />
-        <model name="mobilenet-ssd" precision="FP32" source="omz" />
-        <model name="se-resnext-50" precision="FP32" source="omz" />
-        <model name="caffenet" precision="FP32" source="omz" />
-        <model name="mtcnn-r" precision="FP32" source="omz" />
-        <model name="se-resnet-50" precision="FP32" source="omz" />
-        <model name="densenet-121" precision="FP32" source="omz" />
+        <model name="mobilenet-v2-1.4-224" precision="FP32" source="omz" />
```
Comment (wkobielx) [inline tests/stress_tests/.automation/memcheck_tests/nightly_configs/desktop_test_config.xml:34]: Duplicate mobilenet-v2-1.4-224

File: tests/stress_tests/.automation/memcheck_tests/nightly_configs/desktop_test_config.xml [hunk 2]
```diff
@@ -6,77 +6,75 @@
     <models>
         <!--Models with FP32 precision-->
         <model name="mobilenet-v2-1.4-224" precision="FP32" source="omz" />
-        <model name="brain-tumor-segmentation-0001" precision="FP32" source="omz" />
+        <model name="brain-tumor-segmentation-0002" precision="FP32" source="omz" />
```
Comment (wkobielx) [inline tests/stress_tests/.automation/memcheck_tests/nightly_configs/desktop_test_config.xml:9]: Duplicate brain-tumor-segmentation-0002

**Comment by wkobielx** [comment]
https://github.com/intel-innersource/frameworks.ai.openvino.ci.product-configs/pull/3234 was merged, memleak tests in precommit/postmerge scope should be reenabled now. In case of nightly jobs, need to remove duplicates from yamls.

**Comment by wkobielx** [comment]
@vurusovs @evgenytalanin-intel can you take a look?

**Review by vurusovs — APPROVED** [review]

**Pull request merged**